src/node_crypto.cc

```diff
@@ -741,6 +743,63 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UseExtraCaCerts(const char* file) {
+  free(extra_root_certs_file);
+  extra_root_certs_file = file ? strdup(file) : nullptr;
+}
+
+static void EmitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+  HandleScope handle_scope(env->isolate());
+  Context::Scope context_scope(env->context());
+
+  Local<Object> process = env->process_object();
+  Local<Value> f =
+      process->Get(FIXED_ONE_BYTE_STRING(env->isolate(), "emitWarning"));
```

Comment:
It's this one: https://github.com/nodejs/node/blob/v7.0.0/deps/v8/include/v8.h#L2713